index.html

```diff
@@ -23,16 +23,16 @@ <h4 class="time--stat" style="text-align:center">
       </span>
     </h3>
         <div class="shadow" id="shadow"></div>
-       <center><button class="primary-button" onclick="openModal()" >Start Game</button></center> 
+       <center><button class="primary-button" id="hide" onclick="openModal()" >Start Game</button></center> 
```

Comment:
hide is not a better name, try something like start-btn

Reply:
OK

Reply:
Changed the name from string to level